packages/block-editor/src/components/block-navigation/context.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+export const BlockNavigationContext = createContext( {
```

Comment:
@adamziel I moved the context out into its own file and add a `useBlockNavigationContext` helper.